resources/resincli.robot

```diff
@@ -69,6 +69,28 @@ Device ${device_uuid} log should contain ${value}
     Process ${result}
     Should Contain    ${result.stdout}    ${value}
 
+Add ENV variable Hello with value World to application ${application_name} 
+    ${result} =  Run Process    resin   env     add     Hello    World    -a  ${application_name}
+    Process ${result}
+
+Check if ENV variable Hello with value World is present in application ${application_name} 
+    ${result_vars} =  Run Process    resin envs -a ${application_name} | sed '/ID[[:space:]]*NAME[[:space:]]*VALUE/,$!d'   shell=yes
+    Process ${result_vars}
+    ${result} =  Run Process    echo "${result_vars.stdout}" | grep Hello | cut -d ' ' -f 3    shell=yes
+    Process ${result}
+    Should Match   ${result.stdout}   World
+
+Remove ENV variable Hello with value World from application ${application_name}
+    ${result_vars} =  Run Process    resin envs -a ${application_name} | sed '/ID[[:space:]]*NAME[[:space:]]*VALUE/,$!d'   shell=yes
```

Comment:
Why are we checking that it exits before removing? Resin cli should handle that.